src/runtime/runtimeClient.ts

```diff
@@ -132,7 +133,14 @@ export abstract class RuntimeClient {
       for(let error of buildErrors) {
         error.injectSpan(spans, extraInfo);
       }
-      this.send(JSON.stringify({type: "parse", generation: data.generation, text, spans, extraInfo}));
+      var css = "";
+      this.lastParse.code.replace(/(?:```|~~~)css\n([\w\W]*?)\n(?:```|~~~)/g, (g0, g1) => { // \n excludes disabled blocks
+        css += g1;
+      });
+      css = css ? css.split("\n").map(function(line) {
```

Comment:
Unfortunately not everyone likes to write their CSS with a single line per selector, so we need to handle two other major cases:

``` css
.foo .bar {
  background: red;
  color: green
}
```

and


``` css
.foo .bar
{
  background: red;
  color: green
}
```
I think you can hit all 3 with the following rules:

1. The line begins with a non-brace and contains an opening brace
2. The line begins with a non-brace and is followed by a line containing an open brace prior to a line containing a closing brace.

Once you've matched just selector lines, you also need to split those on commas and inject the selector prefix on each chunk.